src/shell/filesystem_shell.rs

```diff
@@ -559,6 +559,14 @@ impl Shell for FilesystemShell {
             }
         };
 
+        if sources.is_empty() {
+            return Err(ShellError::labeled_error(
+                "Move aborted. Not a valid destination",
+                "not a valid destination",
```

Comment:
I think if sources is empty, the glob on line 538 matched nothing. That means the message and label should perhaps be consistent with other methods. For example:

```
❯ ls *asdljasdluasduklasd
error: Invalid File or Pattern
- shell:1:23
1 | ls *asdljasdluasduklasd
  |    ^^^^^^^^^^^^^^^^^^^^ Invalid File or Pattern
```

One other small nitpick: I would put this above line 549. That way we don't have to do unnecessary work (line 550) if there are no sources 🙂

Reply:
Yep I agree with the change of the error message too.
I made the changes I hope everything is okay!